app/lib/MyAccountsStore.js

```diff
@@ -141,18 +195,34 @@ export default class MyAccountsStore {
   }
 
   fetch (method, url, body) {
+    const STACK_DOMAIN = '//' + document.querySelector('[role=application]').dataset.cozyDomain
```

Comment:
Two point here:
*  We should not rely call document from a lib
*  Fetch may be called by cozy-client-js
However, maybe we could merge as it and integrate cozy-client-js later.